omod/src/main/java/org/openmrs/module/pihcore/page/controller/account/AccountPageController.java

```diff
@@ -46,27 +45,35 @@ public class AccountPageController {
 
     protected final Log log = LogFactory.getLog(getClass());
 
-
-    public PihAccountDomainWrapper getAccount(@RequestParam(value = "personId", required = false) Person person,
-                                              @SpringBean("pihCoreService") PihCoreService pihCoreService) {
+    public PihAccountDomainWrapper getAccount(
+            @RequestParam(value = "personId", required = false) Person person,
+            @SpringBean("pihCoreService") PihCoreService pihCoreService) {
 
         if (person == null) {
             person = new Person();
         }
         return pihCoreService.newPihAccountDomainWrapper(person);
     }
 
-    public void get(PageModel model, @MethodParam("getAccount") PihAccountDomainWrapper account,
+    public String get(PageModel model,
+                    @MethodParam("getAccount") PihAccountDomainWrapper account,
+                    @RequestParam(value = "edit", required = false) Boolean edit,
                     @SpringBean("accountService") AccountService accountService,
                     @SpringBean("adminService") AdministrationService administrationService,
                     @SpringBean("providerManagementService") ProviderManagementService providerManagementService) {
 
         model.addAttribute("account", account);
+        model.addAttribute("editMode", edit == Boolean.TRUE || account.getPerson().getPersonId() == null);
```

Comment:
Now defaults to view mode, but can open in edit mode.  Previously only edit mode.